src/index.js

```diff
@@ -0,0 +1,118 @@
+import ui from './modules/ui';
+import Gameboard from './modules/data/gameboard';
+import Player from './modules/data/players';
+
+   const playerStorage = new Array(100).fill(null);
+   const computerStorage = new Array(100).fill(null);
+
+   const playerBoard = Gameboard();
+   const computerBoard = Gameboard();
+   
+   const player = Player('player', playerBoard, playerStorage);
+   const computer = Player('computer', computerBoard, computerStorage);
+
+   let turn = 'player';
+   
+  const renderBoards = () => {
+   player.showBoard(playerStorage, 'playerBoard');
+   computer.showBoard(computerStorage, 'computerBoard');
+  };
+  
+  const computerInit = () => {
+    const { ships: [carrier, battleship, cruiser, submarine, destroyer ] } = computerBoard
+    const predefinedPlacements = [
+     [ [carrier, 0, computerStorage, 'down'], 
+       [battleship, 2, computerStorage, 'right'], 
+       [cruiser,7 , computerStorage, 'down'], 
+       [submarine, 41, computerStorage, 'right'],
+       [destroyer, 70, computerStorage, 'right'] 
+     ], 
+      [ [carrier, 0, computerStorage, 'down'], 
+      [battleship, 2, computerStorage, 'down'], 
+      [cruiser, 3, computerStorage, 'down'], 
+      [submarine, 4, computerStorage, 'down'],
+      [destroyer, 5, computerStorage, 'down'] 
+     ], 
+     [ [carrier, 9, computerStorage, 'down'], 
+     [battleship, 2, computerStorage, 'right'], 
+     [cruiser, 30, computerStorage, 'down'], 
+     [submarine, 44, computerStorage, 'right'],
+     [destroyer, 55, computerStorage, 'down'] 
+   ]
```

Comment:
**[ OPTIONAL ]:** This logic right here, is taking care of the placement of the ships in sets of pre-defined spaces, this part should ideally be taken into consideration randomly and dynamically checking if the position is valid whether it's vertical or horizontal. It would be nice to see that implementation, not required, but would clean up your code and showcase your skills as a programmer!. It is tots optional no worries.

Reply:
This is still the same, it was optional no worries.